src/colors/conversions.jl

```diff
@@ -0,0 +1,105 @@
+"""
+    channelify(m::AbstractArray{CT,N}) where {CT <: Colorant, N}
+Channelify function.
+
+Input should be in ``WHN`` order ``\\implies (*, batch)``.
+Output will be in the order ``(*, channels, batch)``.
+
+# Examples
+```jldoctest; setup = :(using Images,DiffImages)
+julia> input_size = (16, 16, 2)
+(16, 16, 2)
+
+julia> size(channelify(HSV.(rand(BGR,input_size...))))
+(16, 16, 3, 2)
+```
+"""
+function channelify(m::AbstractArray{CT,N}) where {CT <: Colorant, N}
+    e = eltype(m)
+    m = channelview(m)
+    if e <: AbstractGray
```

Comment:
Can this not be handled by a generic reshape? Those would be no-ops in the remaining cases

Reply:
Yes it is a simple reshape. I'll change it to a reshape then.